cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -534,3 +535,46 @@ For Kerberos-enabled Hadoop clusters:
     rm -rf /yarn/nm/usercache/cdap
 
   Restart CDAP after removing the usercache(s).
+
+.. _cloudera-configuration-highly-available:
+
+Enabling CDAP HA
+----------------
+In addition to having a :ref:`cluster architecture <admin-manual-install-deployment-architectures-ha>`
+that supports HA (high availability), these additional configuration steps need to followed and completed:
+
+CDAP Master
+...........
+- Install multiple instances of the CDAP Master on different nodes.
+- The instances coordinate amongst themselves, setting a leader and followers.
+
+CDAP Router
+...........
+- Install multiple instances of the CDAP Router on different nodes.
+- Sync the configuration files (``cdap-site.xml``, ``cdap-security.xml``) on the nodes.
+- Place a load balancer, if desired, in front of these nodes.
+- The CDAP Router is stateless, and simply routes requests to the appropriate service.
+
+CDAP Kafka
+..........
+- Two properties govern the Kafka setting in the cluster.
```

Comment:
- Using the Cloudera Manager UI, add additional Role Instances of Role "CDAP Kafka Service" to additional machines
- In the CDAP service configuration, adjust the kafka.default.replication.factor parameter as desired.  For an HA setup with 2 CDAP Kafka instances, this must be 2.
Start each CDAP Kafka Service role.